.github/workflows/rust.yml

```diff
@@ -8,15 +8,15 @@ jobs:
 
 
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v2
 
     - name: Install dependencies
       run: |
         sudo apt install -y \
           libpam0g-dev \
-          libssl1.0-dev
+          libudev-dev
```

Comment:
We should consider leaving libssl1.0-dev in place, as a future change to maek some downstream packaging easier I've had to remove bundling by default, now you have to request them. So we probably also need to set `--features=libsqlite3-sys/bundled` in the cargo build line, or add libsqlite dev as well I think